src/pages/de/core-concepts/endpoints.mdx

```diff
@@ -0,0 +1,282 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Endpunkte
+description: Erfahre wie du Endpunkte erstellen kannst, die jegliche Form von Daten bereitstellen
+---
+
+Astro lässt Dich unterschiedliche projektspezifische Endpunkte erstellen, die jegliche Form von Daten bereitstellen können. Du kannst sie verwenden um Bilder zu generieren, ein RSS-Dokument bereitzustellen oder die als API-Route verwenden um eine komplette API für Deine Seite bereitzustellen.
```

Comment:
```suggestion
Astro lässt Dich unterschiedliche projektspezifische Endpunkte erstellen, die jegliche Form von Daten bereitstellen können. Du kannst sie verwenden um Bilder zu generieren, ein RSS-Dokument bereitzustellen oder sie als API-Route verwenden um eine komplette API für Deine Seite bereitzustellen.
```

Ich glaube du wolltest in dem Satz Abschnitt 

> [...], ein RSS-Dokument bereitzustellen oder die als API-Route verwenden [...]

 anstelle von "die" "sie" schreiben, hast dich aber verdrückt.